test/globalize/migration_test.rb

```diff
@@ -263,7 +269,8 @@ def assert_migration_table(fields, model = Migrated)
     assert model.translation_class.index_exists_on?(index_field)
 
     assert_equal :string,   column_type(:locale, model)
-    assert_equal :integer,  column_type(index_field, model)
+    # it's :bigint by default starting from Rails 5
+    assert_equal (Globalize.rails_5? && !Globalize::Test::Database.sqlite?) ? :bigint : :integer, column_type(index_field, model)
```

Comment:
this will start failing from `rails_6?` onward :wink:

Reply:
Oh, right. Didn't think of that. Will fix :)